config.py

```diff
@@ -154,6 +154,7 @@ class Config(object):
                                                   '"US/Central": "US/Central", "US/Eastern": "US/Eastern"}'))
 
     GROUPS_BUILDING_BLOCK_ENDPOINT = os.getenv("GROUPS_BUILDING_BLOCK_ENDPOINT", "https://api-dev.rokwire.illinois.edu/gr/api/int/user/")
+    GROUPS_BUILDING_BLOCK_BASE_URL = os.getenv("GROUPS_BUILDING_BLOCK_BASE_URL", "https://api-dev.rokwire.illinois.edu/gr/api/")
```

Comment:
It's good that you are adding a base URL configuration. It looks like GROUPS_BUILDING_BLOCK_ENDPOINT is only used in one place. Could you please also change it to use the new configuration item?

Reply:
I see that the previous environment variable has been removed. Thanks. 

Going forward, could you please also add a comment in the conversation when a specific item has been addressed? I think It will help when multiple conversations are happening in a PR. Thanks.